src/main/frontend/handler/web/nfs.cljs

```diff
@@ -312,90 +312,94 @@
+                                             ;; Hack until we better understand failure in frontend.handler.file/alter-file
+                                             (global-config-handler/global-config-dir-exists?))
+                                      (p/let [global-files-result (fs/get-files
+                                                                   (global-config-handler/global-config-dir)
+                                                                   (constantly nil))
+                                              global-files (-> (->db-files mobile-native? electron? (global-config-handler/global-config-dir) global-files-result)
+                                                               (remove-ignore-files (global-config-handler/global-config-dir) nfs?))]
+                                        global-files)
+                                      (p/resolved []))
+                   new-files (concat new-local-files new-global-files)
 
-                         _ (when nfs?
-                             (let [file-paths (set (map :file/path new-files))]
-                               (swap! path-handles (fn [handles]
-                                                     (->> handles
-                                                          (filter (fn [[path _handle]]
-                                                                    (contains? file-paths
-                                                                               (string/replace-first path (str dir-name "/") ""))))
-                                                          (into {})))))
-                             (set-files! @path-handles))]
-                        (handle-diffs! repo nfs? old-files new-files handle-path path-handles re-index?))))
-        (p/catch (fn [error]
-                   (log/error :nfs/load-files-error repo)
-                   (log/error :exception error)))
-        (p/finally (fn [_]
-                     (state/set-graph-syncing? false))))))))
+                   _ (when nfs?
+                       (let [file-paths (set (map :file/path new-files))]
+                         (swap! path-handles (fn [handles]
+                                               (->> handles
+                                                    (filter (fn [[path _handle]]
+                                                              (contains? file-paths
+                                                                         (string/replace-first path (str dir-name "/") ""))))
+                                                    (into {})))))
+                       (set-files! @path-handles))]
+             (handle-diffs! repo nfs? old-files new-files handle-path path-handles re-index? ok-handler))))
+       (p/catch (fn [error]
+                  (log/error :nfs/load-files-error repo)
+                  (log/error :exception error)))
+       (p/finally (fn [_]
+                    (state/set-graph-syncing? false)))))))
```

Comment:
Can we change this status name for the re-index UI? 
It's ez to be confused with Logseq Sync